### PR TITLE
[Merged by Bors] - feat: port RingTheory.Polynomial.Cyclotomic.Basic

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean [hunk 1]
```diff
@@ -47,11 +47,8 @@ results hold if there is a primitive `n`-th root of unity in `R`. In particular,
 not the standard one unless there is a primitive `n`th root of unity in `R`. For example,
 `cyclotomic' 3 ℤ = 1`, since there are no primitive cube roots of unity in `ℤ`. The main example is
 `R = ℂ`, we decided to work in general since the difficulties are essentially the same.
-To get the standard cyclotomic polynomials, we use `int_coeff_of_cycl`, with `R = ℂ`, to get a
-polynomial with integer coefficients and then we map it to `R[X]`, for any ring `R`.
-To prove `cyclotomic.irreducible`, the irreducibility of `cyclotomic n ℤ`, we show in
-`cyclotomic_eq_minpoly` that `cyclotomic n ℤ` is the minimal polynomial of any `n`-th primitive root
-of unity `μ : K`, where `K` is a field of characteristic `0`.
+To get the standard cyclotomic polynomials, we use `unique_int_coeff_of_cycl`, with `R = ℂ`,
```
Comment (jcommelin) [inline Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean:50]: It looks like some part of the docstring got lost...

**Review by jcommelin — COMMENTED** [review]
one comment, otherwise lgtm

File: Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean (hunk 1 above)
Comment (Parcly-Taxel) [reply Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean:50]: This part was split off in leanprover-community/mathlib#19144.

Comment (jcommelin) [reply Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean:50]: aah, ok

**Comment by jcommelin** [comment]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by Parcly-Taxel** [comment]
bors merge

**Pull request closed without merging**